models/iosxr/vrf/iosxr_vrf.yaml

```diff
@@ -0,0 +1,124 @@
+            safi:
+              description: Address Family modifier
+              type: str
+              choices: [ 'flowspec', 'multicast', 'unicast']
+            export:
+              description: VRF export
+              type: dict
+            import:
+              description: VRF import
+              type: dict
+            route_policy:
+              description: Use route_policy for export
+              type: str
+            route_target:
+              description: Specify export route target extended communities.
+              type: str
+            to:
+              description: Export routes to a VRF
+              type: dict
+            from:
+              description: Import routes from a VRF
+              type: dict
+            bridge_domain:
+              description: VRF import
+              type: dict
+            advertise_as_vpn:
+              description: Advertise local EVPN imported routes to PEs
+              type: bool
+            default_vrf:
+              description: Export routes to default VRF
+              type: bool
+            vrf:
+              description: Export routes to a VRF
+              type: str
+            allow_imported_vpn:
+              description: Allow export of imported VPN routes to non-default VRF
+              type: bool
+            maximum:
+              description: Set maximum prefix limit
+              type: dict
+              suboptions:
+                prefix:
+                  description:  Set table's maximum prefix limit.
+                  type: int
+                threshold:
+                  description: mid-thresh (% of max).
+                  type: int
+        description:
+          description: Provides a short description of the VRF definition in the current active configuration.
+          type: str
+        evpn_route_sync:
+          description: EVPN Instance VPN ID used to synchronize the VRF route(s).
+          type: int
+        fallback_vrf:
+          description: Fallback VRF name
+          type: str
+        mhost:
+          description: Multicast host stack options
+          type: dict
+          suboptions:
+            afi:
+              description: Address Family Identifier (AFI)
+              type: str
+              choices: ['ipv4', 'ipv6']
+            default_interface:
+              description: Default interface for multicast.
+              type: str
+        rd:
+          description: VPN Route Distinguisher (RD).
+          type: str
+        remote_route_filtering:
+          description: Enable/Disable remote route filtering per VRF
+          type: bool
+        vpn:
+          description: VPN ID for the VRF
+          type: dict
+          suboptions:
+            id:
+              description: VPN ID for the VRF.
+              type: int
+  running_config:
+    description: The state the configuration should be left in.
+      - State I(purged) removes all the VRF configurations from the
```

Comment:
Can you share examples on how state: purged behaves differently from state: deleted?